components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.api.synchronizer/src/main/java/org/wso2/carbon/apimgt/micro/gateway/api/synchronizer/util/APIMappingUtil.java

```diff
@@ -266,19 +266,19 @@ private static void setMaxTpsFromApiDTOToModel(APIDTO dto, API api) {
      * @param apiStatus API status
      * @return API status
      */
-    private static APIStatus mapStatusFromDTOToAPI(String apiStatus) {
+    private static String mapStatusFromDTOToAPI(String apiStatus) {
         // switch case statements are not working as APIStatus.<STATUS>.toString() or APIStatus.<STATUS>.getStatus()
         //  is not a constant
         if (apiStatus.equals(APIStatus.BLOCKED.toString())) {
```

Comment:
Didn't we remove the enum?

Reply:
This is the old code right?

Reply:
the fix is to get rid of the enum right?

Reply:
yes. But your comment shows up in the left pane, which is the older code.

Reply:
I see it as an unchanged code.